charts/datahub/subcharts/datahub-ingestion-cron/Chart.yaml

```diff
@@ -14,8 +14,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.2.2
+version: 0.2.3
```

Comment:
Can you update the main Chart version? charts/datahub/Charts.yaml

No need to update version here

Reply:
Done!